validator/templates/validator/result.html

```diff
@@ -305,6 +359,10 @@ <h4 class="my-0 font-weight-normal">Result files</h4>
     changeOverviewMap()
 </script>
 
+<script type="text/javascript" id="MathJax-script" async
```

Comment:
I imported MathJax for the metric rendering, could this perhaps be connected to the issue somehow @sheenaze ?

Reply:
might be. I'll download it and check if it works if I have it locally.

Reply:
ok, I needed to do some update, now everything looks fine.

Reply:
great! so after merging your changes to the reader by @wpreimes is should be finally good to go, right?